components/services/services.sass

```diff
@@ -8,6 +11,7 @@
 .services__header
   +section-header
   margin-top: -2rem
+  font-weight: 300
```

Comment:
Let's make some mixins for font-weights to make them more readable.

Something like +font-weight-regular +font-weight-light etc.
This way it'll be easy to translate from design documents.


// Font weight
=font-weight-bold
  font-weight: 700

=font-weight-medium
  font-weight: 500

=font-weight-regular
  font-weight: 400

=font-weight-light
  font-weight: 300